nodejs/PhoneNumberOrdering/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "phonenumberordering",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "bandwidth-iris": "^1.0.1",
```

Comment:
super super picky, but can you use https://www.npmjs.com/package/@bandwidth/numbers instead of bandwidth-iris? It's the same package but bundled under our bandwidth org

Reply:
Yea no problem